app.js

```diff
@@ -82,6 +118,50 @@ app.get('/profile', async (req, res)=>{
 
 
 
+//booking api started
+
+
+app.post('/booking', async (req, res)=>{
+  const {username, email, branch, date, deskNo} = req.body;
+
+   await Users.findOne({ email: email}).then((userExist)=>{
+  
+    if(userExist){
+      const bookings = new Booking({username, email, branch, date, deskNo});
```

Comment:
Here you are not handling the case that if the desk is already booked for that particular date and time, then you should give and error response that desk is already booked